### ISSUE TITLE
Update documentation to reflect present tense

### ISSUE THREAD

**Issue by ajsuth** [ordercloud-api/oc-documentation#370]
TLDR; Knowledge-base articles should be current and relevant?

An observation from reviewing many knowledge-base articles is that there is a lot of talk about new and upcoming features. Articles can date back a number of years and lead to questions as to whether they are up-to-date and still relevant.

My suggestion here is that by updating articles where applicable to speak in the present tense we can give the impression that the articles are being maintained and are always current. This also cuts the fat around how functionality previously behaved or lengthy introductions of what is going to be released. [Introducing Premium Search](https://ordercloud.io/knowledge-base/introducing-premium-search), which seems like it may have been migrated from a blog post.

If needed, we could also add caveats to flag changes in previous behaviour and when they were changed, however this could be addressed as they come up.
